src/templates/resources/footer.json

```diff
@@ -568,50 +568,50 @@
         "cuisines": "Top cuisines",
         "cuisineLinks": [
             {
-                "url": "/chinese-takeaways",
+                "url": "/takeaway/nearme/chinese",
                 "text": "Chinese"
             },
             {
-                "url": "/indian-takeaways",
+                "url": "/takeaway/nearme/indian",
                 "text": "Indian"
             },
             {
-                "url": "/italian-takeaways",
+                "url": "/takeaway/nearme/italian",
                 "text": "Italian"
             },
             {
-                "url": "/japanese-takeaways",
+                "url": "/takeaway/nearme/sushi",
```

Comment:
Should it be `/takeaway/nearme/japanese` ?

Reply:
I've updated the text to say "sushi" to match the link